cfg/virtme-ng.conf

```diff
@@ -1,4 +1,11 @@
 {
-    "default_opts" : {
+    "default_opts": {},
+    "systemd": {
+        "masks": [
```

Comment:
If I'm not mistaken, the corresponding local config file will not be updated during the upgrade. I even think that this default config file might not even be installed, e.g. when `virtme-ng` is installed with `pip` or other Python installers. So if this is correct, I guess you will need to handle that case, e.g. the new `get_conf()` should return a default value given in argument.

Reply:
Added a commit addressing the config: https://github.com/arighi/virtme-ng/pull/280/commits/95158951d1bbe11971755083837e58ba177e2a55